### DISCUSSION TITLE
How can I `neotest.output.open()` test result that's not on my cursor?

### DISCUSSION THREAD

**Discussion by ziontee113** [nvim-neotest/neotest#194] [Q&A] [answered]
Hi! I'm currently using `neotest-plenary`, and I'm stumbling on this use case. 
I'm using this mapping:

```lua
        vim.keymap.set("n", "to", function()
            neotest.output.open({ enter = true, short = true }) -- output window
        end, {})
```

When I `print` stuffs with Lua, I can see those prints if I use the mapping only on  `describe` blocks, not the `it` blocks. It'd be convenient if I can just open the test results of the `describe` block without having to scroll up to it. 

Is there already a solution for this in `neotest`? Thank you very much!

**Comment by rcarriga** [comment, marked as answer]
The `short` message is supplied directly from plenary so neotest-plenary can't do anything to change it. The message is just the error that is raised, it's not actually capturing output so I'd say trying to implement this would be a large piece of work.

You can of course not use the `short` output but it won't just show the specific message for the selected test

**Reply by ziontee113** [reply]
Thank you very much for the response. I'll probably gonna have 2 different keymaps for `output.open()` with `short` and without it :+1:
